app/views/supervisors/dashboard/index.html.slim

```diff
@@ -43,7 +43,7 @@ div.row
             p = "You last checked on them #{ time_ago_in_words(t.comments.last.created_at) } ago."
           - else
             p You didn't check on them yet.
-          = simple_form_for Comment.new(commentable: t), url: supervisor_comments_path do |f|
+          = simple_form_for Comment.new(commentable: t), url: supervisors_comments_path do |f|
```

Comment:
@F3PiX I think you also need to change `supervisor_comments_path` to `supervisors_comments_path` on line 50.